lib/gringotts/gateways/mercadopago.ex

```diff
@@ -0,0 +1,376 @@
+  | ------                       | ------        | 
+  | Pre-authorize                | `authorize/3` | 
+
+  [home]: https://www.mercadopago.com/
+  [docs]: https://www.mercadopago.com.ar/developers/en/api-docs/
+
+  ## The `opts` argument
+
+  Most `Gringotts` API calls accept an optional `keyword` list `opts` to supply
+  optional arguments for transactions with the MERCADOAPGO
+  gateway. The following keys are supported:
+
+  | Key                      | Remark                                                                                  |
+  | ----                     | ---                                                                                     |
+  | `email`                  | Email of the customer. Type - string .                                                  |
+  | `order_id`               | Order id issued by the merchant. Type- integer.                                         |
+  | `payment_method_id`      | Payment network operators, eg. `visa`, `mastercard`. Type- string.                      |
+  | `customer_id`            | Unique customer id issued by the gateway. For new customer it must be nil. Type- string |  
+
+  ## Registering your MERCADOPAGO account at `Gringotts`
+
+  After [making an account on MERCADOPAGO][credentials], head to the credentials and find
+  your account "secrets" in the `Checkout Transparent`.
+
+  | Config parameter | MERCADOPAGO secret |
+  | -------          | ----               |
+  | `:access_token`  | **Access Token**   |
+  | `:public_key`    | **Public Key**     |
+  
+  > Your Application config **must include the `[:public_key, :access_token]` field(s)** and would look
+  > something like this:
+  > 
+  >     config :gringotts, Gringotts.Gateways.Mercadopago,
+  >         public_key: "your_secret_public_key"
+  >         access_token: "your_secret_access_token"
+
+  [credentials]: https://www.mercadopago.com/mlb/account/credentials?type=basic
+
+  * MERCADOPAGO does not process money in cents, and the `amount` is rounded to 2
+    decimal places.
+
+  > Please [raise an issue][new-issue] if you'd like us to add support for more
+  > currencies
+  [new-issue]: https://github.com/aviabird/gringotts/issues
+
+  ## Supported currencies and countries
+
+  > MERCADOPAGO supports the currencies listed [here][all-currency-list]
+  [all-currency-list]: https://www.mercadopago.com.br/developers/en/api-docs/account/payments/search
+      :ARS, :BRL, :VEF,:CLP, :MXN, :COP, :PEN, :UYU
+
+  ## Following the examples
+
+  1. First, set up a sample application and configure it to work with MERCADOPAGO.
+  - You could do that from scratch by following our [Getting Started][gs] guide.
+      - To save you time, we recommend [cloning our example
+      repo][example] that gives you a pre-configured sample app ready-to-go.
+          + You could use the same config or update it the with your "secrets"
+          as described [above](#module-registering-your-monei-account-at-mercadopago).
```

Comment:
monei??

Reply:
This is because of a bug in the template file (`templates/gateway.eex`) used by `gringotts.new`. Could I interest you in submitting a tiny PR with the fix for #136?